pkg/p2p/libp2p/libp2p.go

```diff
@@ -60,14 +60,15 @@ type Service struct {
 }
 
 type Options struct {
-	PrivateKey  *ecdsa.PrivateKey
-	NATAddr     string
-	DisableWS   bool
-	DisableQUIC bool
-	LightNode   bool
-	Addressbook addressbook.Putter
-	Logger      logging.Logger
-	Tracer      *tracing.Tracer
+	PrivateKey     *ecdsa.PrivateKey
+	NATAddr        string
+	DisableWS      bool
+	DisableQUIC    bool
+	LightNode      bool
```

Comment:
@janos, we don't have light node capabilities atm. Why is this `bool` here?

Reply:
It is added because the handshake defines it. The feature is still not there but the protocol specifications has it and with this flag it is configurable.